### ISSUE TITLE
yoshi-config/jest errors are not informative enough

### ISSUE THREAD

**Issue by rokasmik** [wix/yoshi#1234]
# 🐛 Bug Report

### 😯 Current Behavior

When running `yoshi test` with `jest` and parsing `jest-yoshi.config.js`, the compilation errors do not display a stacktrace of where the error has happened. 
E.G.
```
Error: Config /Users/test/git/my-app/jest-yoshi.config.js is invalid:
  (0 , _reactI18next.translate) is not a function
    at loadConfig (/Users/test/git/my-app/node_modules/yoshi-config/jest.js:22:11)
...
```
This does not help to detect where the issue is happening

### 🤔 Expected Behavior
Original error stack trace should be displayed, same example:
```
TypeError: (0 , _reactI18next.translate) is not a function
    at Object.<anonymous> (/Users/test/git/my-app/node_modules/some-module/i18next.js:59:51)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
...
```

### 💁 Possible Solution
log error stack trace, not only message `yoshi-config/jest.js` where `throw new Error('Config ${chalk.bold(configPath)} is invalid:\n  ${error.message}\');` is done

### 🌍 Your Environment
yoshi 4.2.2

**Comment by ranyitz** [comment]
@rokasmik Nice! It looks like this small fix could spare a lot of time to devs that encounter this problem :) Can you submit a PR that fixes it?

**Comment by ranyitz** [comment]
Adding a link to the specific line:

https://github.com/wix/yoshi/blob/9fd5883caef273255962be82defc6505ca7a08f6/packages/yoshi-config/jest.js#L22

**Comment by rokasmik** [comment]
@ranyitz will do, couldn't get my hands on it :/

**Comment by rokasmik** [comment]
Created a PR https://github.com/wix/yoshi/pull/1251

**Comment by ranyitz** [comment]
Fixed on #1251
